tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -70,17 +68,18 @@ jobs:
 
   - script: |
       set -e -x
-      cd android_test/android
-      appcenter test run espresso \
-        --app "AI-Frameworks/ORT-Mobile-Android" \
-        --devices $(app_center_android_test_devices) \
-        --app-path ./app/build/outputs/apk/debug/app-debug.apk \
-        --test-series "master" \
-        --locale "en_US" \
-        --build-dir ./app/build/outputs/apk/androidTest/debug \
-        --token $(app_center_api_token)
-    displayName: Run E2E tests using App Center
-    workingDirectory: $(Build.BinariesDirectory)
+      cd tools/python
+      pip install requests
+      export BROWSERSTACK_ID=$(browserstack_username)
+      export BROWSERSTACK_TOKEN=$(browserstack_access_key)
```

Comment:
do secrets need to be set via the `env` property?

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#use-a-secret-variable-in-the-ui

Reply:
ohh this is much cleaner